public/locales/en/app.ftl

```diff
@@ -0,0 +1,304 @@
+scan-submit = Scan
+scan-another-email = Scan Another Email Address
+scan-featuredbreach-label = Find out if your <span class="featured-breach-title"> --- </span> account was compromised.
+scan-error = Must be a valid email.
+
+signup-banner-headline = Stay safe with {-product-name} protection.
+signup-banner-blurb = Sign up for {-product-name}. You’ll get a full report on your compromised accounts and notifications any time your accounts appear in new data breaches.
+signup-button = Sign up
+
+download-firefox-bar-blurb = {-product-name} is brought to you by the all-new {-brand-name}.
+download-firefox-bar-link = Download {-brand-name} now
+
+download-firefox-banner-blurb = Take control of your browser
+download-firefox-banner-button = Download {-brand-name}
+
+signup-modal-headline = Sign Up for {-product-name}
+signup-modal-blurb = Sign up for your full report, alerts when new breaches happen, and safety tips from {-product-name}.
+signup-modal-terms-and-privacy = Terms and Privacy
+signup-modal-close = Close
+
+signup-modal-verify-headline = Verify Your subscription
+signup-modal-verify-blurb = We sent a verification link to <span id="submitted-email" class="medium"></span>.
+signup-modal-verify-expiration = This link expires in 24 hours.
+signup-modal-verify-resend = Not in inbox or spam folder? Resend.
+
+signup-with-fxa - Sign Up with {-brand-name} Account
+
+form-signup-placeholder = Enter email
+form-signup-checkbox = Get the latest from {-brand-Mozilla} and {-brand-name}.
+form-signup-submit = Sign up
+form-signup-error = Must be a valid email.
+
+no-breaches-headline = So far, so good
+no-breaches-blurb = Your account did not appear in our basic scan. That's good news, but data breaches can happen any time and there is still more you can do. Subscribe to {-product-name} for a full report, alerts when new breaches happen, and tips on protecting your passwords.
+
+found-breaches-headline = This might be a problem.
+found-breaches-singular-blurb = Your email address was compromised in the following breach.
+found-breaches-plural-blurb = Your email address was compromised in the following breaches.
+
+show-more-breaches = Show More
+
+wtd-section-headline = What To Do
+wtd-1 = Change your password on these sites and anywhere else you've used the same password.
+wtd-2 = Make your answers to security questions just as strong as your passwords.
+wtd-3 = Use password managers like 1Password, LastPass, or Dashlane to generate strong passwords, remember them, and fill them into websites so you don't have to.
+wtd-4 = Subscribe to alerts from {-product-name} to learn sooner about your compromised accounts.
+
+
+featured-breach-only = Your email address appears in the <span class="featured-breach-title"> --- </span> breach, but does not appear in any other known breaches.
+
+featured-breach-results = Your email address appears in the { $featuredBreachTitle } breach, { $breachCount ->
+  [1] one other breach.
```

Comment:
Do we really need to spell out numbers up to ten? Do we have data on the average number of breaches?

Reply:
It [does](https://pontoon.mozilla.org/en-CA/firefox/browser/browser/preferences/preferences.ftl/?search=%5B1%5D&string=176248). But I'm almost sure the UI would be useful with this many variants, and they would be automatically exposed to all languages, instead of a standard plural form (we should verify it somehow).